src/main/java/se/liu/ida/hefquin/cli/RunQueryWithoutSrcSel.java

```diff
@@ -158,9 +160,18 @@ protected void exec() {
 		}
 
 		if (    statsAndExceptions != null
-		     && statsAndExceptions.object1 != null
-		     && contains(argQueryProcStats) ) {
-			StatsPrinter.print(statsAndExceptions.object1, System.err, true);
+		     && statsAndExceptions.object1 != null) {
+			if ( contains(argQueryProcStats) ) {
+				StatsPrinter.print(statsAndExceptions.object1, System.err, true);
+			}
+			if ( contains(argOnelineTimeStats) ) {
+				final long overallQueryProcessingTime = statsAndExceptions.object1.getOverallQueryProcessingTime();
+				final long planningTime = statsAndExceptions.object1.getPlanningTime();
+				final long compilationTime = statsAndExceptions.object1.getCompilationTime();
+				final long executionTime = statsAndExceptions.object1.getExecutionTime();
+				final String queryProcStats = overallQueryProcessingTime + ", " + planningTime +", " + compilationTime + ", " + executionTime + "\n";
+				System.out.println(queryProcStats);
```

Comment:
Just one last thing: You don't need the line break ("\n") at the end of the string because you are using `println(...)` in the next line. Hence, with the current code you may actually get an empty line in between each CSV line.

Here is the fix for this issue, which I will apply myself right away.
```suggestion
				final String queryProcStats = overallQueryProcessingTime + ", " + planningTime +", " + compilationTime + ", " + executionTime + "\n";
				System.out.println(queryProcStats);
```